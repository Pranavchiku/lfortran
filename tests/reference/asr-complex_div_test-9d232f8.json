{
    "basename": "asr-complex_div_test-9d232f8",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/complex_div_test.f90",
    "infile_hash": "0bbb7100f12786a5fc5022ab6cfd9f97cc6ede016ed0bdab03eda670",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-complex_div_test-9d232f8.stdout",
    "stdout_hash": "cf04b6dc1c5983f5c2c79f32d753ab9c2114665535cf3d0a90d45882",
    "stderr": "asr-complex_div_test-9d232f8.stderr",
    "stderr_hash": "86bcf50551c4187cf9e3d86fc13ce1c2d0ed44d7a5e25fd5cfd85aff",
    "returncode": 0
}