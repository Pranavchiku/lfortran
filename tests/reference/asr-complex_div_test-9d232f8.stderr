warning: Implied save feature
 --> tests/../integration_tests/complex_div_test.f90:2:1
  |
2 | complex :: x
  | ^^^^^^^^^^^^ Assuming save attribute implicitly for variables
